src/libstd/sys/windows/net.rs

```diff
@@ -168,7 +182,6 @@ impl Socket {
                 n => Ok(Socket(n)),
             }
         }?;
-        socket.set_no_inherit()?;
```

Comment:
I think this is still required for accepted sockets?

Reply:
My understanding is that `accept` will inherit the properties of the listening socket, so assuming it was created by rust, it it already configured to not be inherited by the child processes.

I can restore the call though!

Reply:
Hm ok, that seems plausible (I'm not sure myself). Can you run some tests to determine if this is true?

Reply:
I'll try to work something out. I suppose this would preferably be a unit test?

Reply:
Although on a second thought I'm not entirely sure testing sockets works as a unit test per-se

Reply:
It sounds like Alex was asking you to write some code that verifies the behaviour under discussion in order to resolve the discussion, rather than asking you to add automated tests for it to this PR.

Reply:
Er sorry yes, that's what I meant about writing tests. Could you write a test (not necessarily checked in, just verified locally), that if you accept a socket it still isn't inherited by default?